dm/master/config.go

```diff
@@ -65,6 +66,7 @@ func NewConfig() *Config {
 	fs.StringVar(&cfg.InitialCluster, "initial-cluster", "", fmt.Sprintf("initial cluster configuration for bootstrapping, e,g. dm-master=%s", defaultPeerUrls))
 	fs.StringVar(&cfg.PeerUrls, "peer-urls", defaultPeerUrls, "URLs for peer traffic")
 	fs.StringVar(&cfg.AdvertisePeerUrls, "advertise-peer-urls", "", `advertise URLs for peer traffic (default "${peer-urls}")`)
+	fs.StringVar(&cfg.Join, "join", "", `join to an existing cluster (usage: cluster's "${advertise-client-urls}"`)
```

Comment:
seems don't have config named `advertise-client-urls`

Reply:
It's a but, `join` should be the address of clients (endpoints), but use peer address before, fixed in https://github.com/pingcap/dm/pull/350/commits/b0f53328e849e95757c15fe7f599e7115b04258e.